### DISCUSSION TITLE
Ghostty does not open the highest priority config file on MacOS when one exists

### DISCUSSION THREAD

**Discussion by MauriceElliott** [ghostty-org/ghostty#3952] [Feature Requests, Ideas] [answered]
as per issue against the website: https://github.com/ghostty-org/website/issues/177

When a configuration file exists in ``$HOME/Library/Application\ Support/com.mitchellh.ghostty/config``, Ghostty should open it when the settings button is pressed from the ribbon menu or ``cmd+,`` rather than the configuration file in ``$XDG_CONFIG_HOME/ghostty/config``, as it is lower priority.

**Comment by mitchellh** [comment, marked as answer]
Accepted #3953
